### ISSUE TITLE
I convered real estate bot into 2.0 but I'm facing isuues, rasa shell command opens rasa intecative why? opens the detailed confidence information etc.

### ISSUE THREAD

**Issue by vivek22122014** [cedextech/rasa-chatbot-templates#8]
Hello Every time I run “rasa shell” , it runs as if I type “rasa shell nlu” . I am trying to run “rasa shell” normally but it opens the detailed confidence information etc.
![Screenshot (277)](https://user-images.githubusercontent.com/58829538/113283488-fe918000-9305-11eb-9c46-eb650f74c3c2.png)
![Screenshot (278)](https://user-images.githubusercontent.com/58829538/113283491-ffc2ad00-9305-11eb-9646-e2b7b1696802.png)

**Comment by shreeraj04** [comment]
Hi Vivek,
As I can see in your 2nd snapshot the domain file is invalid. Can you figure it out the issue in domail yml file. I guess because of this the core model wasn't trained due to which you are facing the above scenario. 

You can validate your yml file in online yaml validator.

**Comment by vivek22122014** [comment]
> 
> 
> Hi Vivek,
> As I can see in your 2nd snapshot the domain file is invalid. Can you figure it out the issue in domail yml file. I guess because of this the core model wasn't trained due to which you are facing the above scenario.
> 
> You can validate your yml file in online yaml validator.

Yeah I'm facing issues regarding forms in rasa when converting from real estate rasa 1.0 to rasa 2.0 
Can you please help me, I'll send you my project files

**Comment by vivek22122014** [comment]
Thanks for reply @shreeraj04 :)

**Comment by shreeraj04** [comment]
Hi @vivek22122014 Hope you are able to solve it
